src/main/java/net/minecraftforge/installer/SwingUtil.java

```diff
@@ -0,0 +1,27 @@
+package net.minecraftforge.installer;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+
+public class SwingUtil {
+    public static JButton createlogButton() {
```

Comment:
```suggestion
    public static JButton createLogButton() {
```
Lowercase L in the user-facing "Open log" string, however still uppercase L for the method name as it's a separate word from create (camelCase). For example:
- do thing -> doThing()
- create log button -> createLogButton()